Makefile

```diff
@@ -9,8 +9,17 @@ PARALLEL_PROOF_LIMIT := 1
 help: ## Display this help message
 	@awk 'BEGIN {FS = ":.*?## "} /^[a-zA-Z_-]+:.*?## / {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}' $(MAKEFILE_LIST)
 
+.PHONY: build-risc0
+build-risc0:
+	cargo risczero build --manifest-path guests/risc0/batch-proof-bitcoin/Cargo.toml
+	cargo risczero build --manifest-path guests/risc0/light-client-proof-bitcoin/Cargo.toml
```

Comment:
Seems like the build command has `--target-dir` to specify where the elfs should go... but i couldn't find it when running `cargo risczero build` locally.
https://github.com/risc0/risc0/blob/v1.1.3/risc0/cargo-risczero/src/commands/build.rs#L56

Seems this is unreleased yet?

Reply:
are you sure you're on 1.1.3?